clang/include/clang/Driver/Driver.h

```diff
@@ -297,6 +297,9 @@ class Driver {
   /// Object that stores strings read from configuration file.
   llvm::StringSaver Saver;
 
+  /// Linker inputs originated from configuration file.
+  std::unique_ptr<llvm::opt::InputArgList> CfgLinkerInputs;
```

Comment:
`CfgLinkerInputs` is not a good name as the implementation is generic, not coupled with linker options.